catalog/spatial/kml/spatial-kml-util/src/main/java/org/codice/ddf/spatial/kml/converter/MetacardToKml.java

```diff
@@ -111,7 +115,7 @@ private static Geometry createPointGeo(Point jtsPoint) {
     return KmlFactory.createPoint().addToCoordinates(jtsPoint.getX(), jtsPoint.getY());
   }
 
-  private static com.vividsolutions.jts.geom.Geometry readGeoFromWkt(final String wkt)
+  public static com.vividsolutions.jts.geom.Geometry createJtsGeoFromWkt(final String wkt)
```

Comment:
✏️ Should be able to write positive & negative tests for this newly public method now.

Reply:
Added Tests